06.bowling_object/frame.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+require './shot'
+
+class Frame
+  attr_reader :shots, :idx
```

Comment:
```suggestion
  attr_reader :shots
```

外部に公開する必要がないならidxはなくしましょう